go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift-kni/debug-tools
 
-go 1.18
+go 1.20
```

Comment:
this should be kept in the base PR from which this one was extracted. We should only bump the tooling in this PR.

Reply:
There should be a PhD in version bumping